docker-compose.yml

```diff
@@ -13,10 +13,9 @@ services:
     volumes:
         - ${GETH_HOST_DATA_DIR}:/data
     env_file:
-      # select your network here:
-#      - .env.sepolia
-#      - .env.mainnet
-  node:
+        - .env    
+
+  lisk-node:
```

Comment:
do we need to change here?

Reply:
I highlighted in the `readme` to copy network specific `.env.*` file to `.env`. With this change we don't need to update `docker-compose.yml` file every time when we run the node.

Reply:
Updated!